docs/dgfit/dustgrain_props.rst

```diff
@@ -0,0 +1,108 @@
+#####################
+Dust grain properties
+#####################
+
+There are several plots that show the effective grain properties of the used dustgrains.
+In all plots, it is possible to transform the particles and data to the observed data that was used.
+
+To plot the extinction coefficients, albedo, scattering phase function and emission of the used dustgrains (default is astro-silicates) in function of the grain size, use command
+
+    dgplot_dustgrains
+
+.. plot::
+
+    from dgfit.dustgrains import DustGrains
+    from dgfit.plotting.plot_dustgrains import plot
+
+    DG = DustGrains()
+    DG.from_files("astro-silicates", "../../dgfit/data/indiv_grain/")
+
+    plot(DG, 'astro-silicates')
+
+To see other dustgrains (<possible> = astro-silicates, astro-carbonaceous, astro-graphite, astro-PAH-ionized and astro-PAH-neutral), use
+
+    dgplot_dustgrains -c <possible>
+
+To transform the particles to the observed data grids and see the data of the dustgrains for the observed dustmodel, use
+
+    dgplot_dustgrains --obsdata obsdata
+
+To see the options for saving the plots (for this and future plots), use
+
+    dgplot_dustgrains --help
+
```

Comment:
It would be really nice to have a divider here to make it clear that this is a different command/plot.

Maybe add something like:

```
Effective Size
--------------
```

Reply:
Same comment applies whenever a new plot function is used.